docs/graph/authentication/ProviderManagerChangedState.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
For these small enum types, they'll all be auto-generated in the future as part of our .NET docs. We shouldn't create a bunch of files for these in our main docs. Main docs here are for general overall guidance about how the pieces fit together. If you think we should pre-emptively document them, then put them in a parent doc along with where they're most commonly used for now?

Reply:
Should I simply remove them, or integrate the docs for them into whatever related doc page? Like putting ProviderState in the IProvider or ProviderManager doc. Thoughts?

Reply:
Yeah, anything like that would be good where there's more guidance content. If you look at the new doc template we remove all the property tables, so once we generate docs from a NuGet package all those can just go away anyway, so they're not too important to have individual pages for here.